### PR TITLE
Convert LOOC/whisper to TGUI

### PR DESCRIPTION
**Pull request by lessthnthree**
## About The Pull Request

Finishes off the conversion of our say features to TGUI, the last two holdouts LOOC and whisper.

## How This Contributes To The Skyrat Roleplay Experience

Consistent 'say' UI experience.

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>

https://github.com/Skyrat-SS13/Skyrat-tg/assets/83487515/1aa69a6b-87dc-42f3-a236-691cd268d289

</details>

## Changelog

:cl: LT3
refactor: LOOC and whisper channels converted to TGUI
/:cl:

### CODE REVIEW

**Comment by strandsofivy** [comment]
whisper purple is barely visible or is it video compression? also meow

File: tgui/packages/tgui-say/styles/colors.scss [hunk 1]
```diff
@@ -12,7 +12,9 @@ $lightHover: #eaeaea;
 $say: #a4bad6;
 $radio: #1ecc43;
 $me: #5975da;
+$whis: #300098; // SKYRAT EDIT ADDITION - CUSTOMIZATION
```
Comment (tf-4) [inline tgui/packages/tgui-say/styles/colors.scss]: yeahhh this isn't really visible in the provided video

**Review by tf-4 — CHANGES_REQUESTED** [review]

File: tgui/packages/tgui-say/styles/colors.scss (hunk 1 above)
Comment (lessthnthree) [reply tgui/packages/tgui-say/styles/colors.scss]: looked lighter in the vsc preview, turns out TGUI puts a filter or something

**Comment by lessthnthree** [comment]
lightened it up

**Review by tf-4 — APPROVED** [review]

**Pull request merged**